app/controllers/assignments_controller.rb

```diff
@@ -190,9 +188,15 @@ def start_upload(assignment, submissionFile, isMapEnabled, mapFile)
 
       # Process upload file
       submissions_path = SubmissionsHandler.process_upload(submissionFile, isMapEnabled, mapFile, assignment)
+      if submissions_path
+        # Launch java program to process submissions
+        SubmissionsHandler.process_submissions(submissions_path, assignment, isMapEnabled)
```

Comment:
If we later switch to AJAX based updating, can we have submissions push their completion automatically?  This is really a wish list, so really an `enhancement` that we can add this as a enhancement issue later on?